data/mods/gen1/moves.ts

```diff
@@ -881,6 +881,7 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		inherit: true,
 		desc: "Deals Normal-type damage. If this move was successful, the user takes damage equal to 1/2 the HP lost by the target, rounded down, but not less than 1 HP. This move is automatically used if none of the user's known moves can be selected.",
 		shortDesc: "User loses 1/2 the HP lost by the target.",
+		selfDamage: "Recoil",
```

Comment:
```suggestion
```

This should be filled in by the Move constructor, probably. Otherwise it's a lot of redundant data.